src/pages/index.tsx

```diff
@@ -8,9 +16,11 @@ const Home = () => (
       <meta name="viewport" content="width=device-width, initial-scale=1" />
       <link rel="icon" href="/favicon.ico" />
     </Head>
-    <main>
-      <h1 className="text-2xl">Hello HSM!</h1>
-    </main>
+    <HeaderHomePage />
+    <WhoWeAreHomePage />
+    <ProjectsHomePage />
+    <FormatE1HomePage />
+    <PartnershipHomePage />
```

Comment:
I would suggest starting with `HomePage(...)` in those components names so they keep easier to reason about from first word